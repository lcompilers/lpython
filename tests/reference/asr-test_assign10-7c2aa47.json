{
    "basename": "asr-test_assign10-7c2aa47",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_assign10.py",
    "infile_hash": "2fdefb7e44ef02e06e8588641290c812f6837d38bec282e7f9836b16",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_assign10-7c2aa47.stderr",
    "stderr_hash": "26d75468ada03d7e4d454313318d2514847d9a3aa025fc66e10cf2a1",
    "returncode": 2
}