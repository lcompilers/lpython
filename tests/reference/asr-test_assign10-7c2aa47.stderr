semantic error: 'f64' object does not support item assignment
 --> tests/errors/test_assign10.py:6:5
  |
6 |     i[0] = 3.1415926359
  |     ^^^^^^^^^^^^^^^^^^^ 
